### PR TITLE
Telliot Conditional Reporting (tellor is stale / max price change)

### PR DESCRIPTION
**Pull request by 0xSpuddy**
### Steps Taken to QA Changes
Local tests passing, and lots of sepolia testing.

This pull request is:
- [x] A feature implementation

Where Tellor is used as a primary oracle, `telliot contitional` can be used as the reporter of last resort for minimum liveness / heartbeat. Additionally, a price change threshold can be set so that if telliot calculates a value that is a set percentage different from the current oracle value, a new value is automatically reported.

### CODE REVIEW

File: src/telliot_feeds/cli/commands/conditional.py [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+            return
+
+        # Make sure current account is unlocked
+        if not account.is_unlocked:
+            account.unlock(password)
+
+        click.echo("Reporter settings:")
+        click.echo(f"Max tolerated price change: {percent_change * 100}%")
+        click.echo(f"Value considered stale after: {stale_timeout} seconds")
+        click.echo(f"Transaction type: {tx_type}")
```
Comment (akremstudy) [inline src/telliot_feeds/cli/commands/conditional.py:105]: duplicate

File: src/telliot_feeds/cli/commands/conditional.py [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+    pass
+
+
+@conditional_reporter.command()
+@common_options
+@common_reporter_options
+@click.option(
+    "-pc",
+    "--percent-change",
+    help="Price change percentage for triggering a report. Default=0.01 (1%)",
```
Comment (akremstudy) [inline src/telliot_feeds/cli/commands/conditional.py:34]: default 50% comment

File: tests/conftest.py [hunk 3]
```diff
@@ -159,7 +159,7 @@ def mumbai_test_cfg():
 
 @pytest.fixture(scope="function", autouse=True)
 def goerli_test_cfg():
```
Comment (akremstudy) [inline tests/conftest.py:161]: should this still say goerli?

**Review by akremstudy — APPROVED** [review]
left comments on minor things, otherwise lgtm. thanks, @0xSpuddy

File: src/telliot_feeds/reporters/customized/conditional_reporter.py [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+        data, status = await self.oracle.read("getDataBefore", self.datafeed.query.query_id, current_time())
+        if not status.ok:
+            logger.warning(f"error getting tellor data: {status.e}")
+            return None
+        return GetDataBefore(*data)
+
+    async def get_telliot_feed_data(self, datafeed: DataFeed[Any]) -> Optional[float]:
+        """Fetch spot price data from API sources and calculate a value
+        Returns:
+        - Optional[GetDataBefore]: latest data from tellor oracle
```
[the remark this answers is not in the document]
Comment (0xSpuddy) [reply src/telliot_feeds/reporters/customized/conditional_reporter.py:61]: Was grabbing current data, but`liquity backup` used getDataBefore, then I used the same so I could use the same typing. Any advantage to using getCurrentData?

**Pull request merged**